projects/angular/src/accordion/stepper/stepper-panel.spec.ts

```diff
@@ -22,11 +23,18 @@ import { ClrStepperModule } from './stepper.module';
 @Component({
   template: `
     <form clrStepper [formGroup]="form">
-      <clr-stepper-panel formGroupName="groupName">test step</clr-stepper-panel>
+      <clr-stepper-panel
+        [clrAccordionPanelHeading]="showHeading"
+        [clrAccordionPanelHeadingLevel]="headingLevel"
```

Comment:
You might be able to `override` the properties and set the value on the `super` property to just change the `@Input()` alias for the stepper. That would be preferable so that the accordion input names don't even work on stepper panels.

Reply:
I'll remove the test.

Reply:
Corrected in commit: cc43f46